doc/whats-new.rst

```diff
@@ -21,9 +21,38 @@ v0.9.7 (unreleased)
 Enhancements
 ~~~~~~~~~~~~
 
+- Support for `pathlib.Path` objects added to
```

Comment:
nit: we typically cite the issue number (e.g. `:issue: 799:`). Would be nice to include here.

Reply:
I just pushed a commit to add this